.github/workflows/main.yml

```diff
@@ -2,20 +2,39 @@ name: Docker image CI
 
 on:
   push:
+    # takes muliple branch names
     branches: ['master']
-  workflow_dispatch:
+  
+env:
+  IMAGE_TAG: latest
+  REPO_URL: ${{ github.repository }}
+    
 jobs:
   build:
+    # to test a feature, change the repo name to your github id
+    if: github.repository_owner == 'tl-its-umich-edu'
```

Comment:
This way we could stop the build for forked branches. I see an action recored momentarily and it skips  https://github.com/pushyamig/my-learning-analytics/actions/runs/6000238741

The better way if we want to skip it completely, that we need to create github PAT https://github.com/peter-evans/repository-dispatch. I think i will skip that far at the moment and keep researching the best way

Reply:
I think this is good for now.